ambari-server/src/main/python/ambari_server_main.py

```diff
@@ -60,6 +60,7 @@
     "-XX:+UseConcMarkSweepGC " + \
     "-XX:-UseGCOverheadLimit -XX:CMSInitiatingOccupancyFraction=60 " \
     "-Dsun.zip.disableMemoryMapping=true " + \
+    "-Djava.security.egd=file:/dev/./urandom " + \
```

Comment:
Although it might be true that reading from /dev/urandom is faster since it is non-blocking, we must understand that it is non-blocking because it's not doing the work to generate entropy for randomness. If the use of /dev/random is blocking, this means that the entropy pool is drained and it needs to generate more. 

Since any customer can do this on their own, I would rather not make this a product change since it could have security implications.